### DISCUSSION TITLE
Modify post "Apache Solr with Drupal 8 and Search API Solr"

### DISCUSSION THREAD

**Discussion by KarimBoudjema** [ddev/ddev#2719] [General]
The post "Apache Solr with Drupal 8 and Search API Solr" in https://www.ddev.com/ddev-local/apache-solr-with-drupal-8-and-search-api-solr/ has 2 typos:

1. 
// Get the docker-compose.solr.yaml
cd .ddev && curl -O https://raw.githubusercontent.com/drud/ddev/master/pkg/servicetest/testdata/services/docker-compose.solr.yaml

**should be**

// Get the docker-compose.solr.yaml
cd .ddev && curl -O https://raw.githubusercontent.com/drud/ddev/master/pkg/servicetest/testdata/TestServices/docker-compose.solr.yaml

(in the URL replace _services_ with _TestServices_)

2.
// Add the solr-configupdate.sh script
mkdir -p solr/conf
cd solr && rm -rf solr-configupdate.sh && curl -O https://raw.githubusercontent.com/drud/ddev/master/pkg/servicetest/testdata/services/solr-configupdate.sh && chmod +x solr-configupdate.sh

**should be**

mkdir -p solr/conf
cd solr && rm -rf solr-configupdate.sh && curl -O https://raw.githubusercontent.com/drud/ddev/master/pkg/servicetest/testdata/TestServices/solr-configupdate.sh && chmod +x solr-configupdate.sh

(in the URL replace _services_ with _TestServices_)

BTW, this is a great post and I really recommend it!

**Comment by rfay** [comment]
Thanks so much! Fixed in https://www.ddev.com/ddev-local/apache-solr-with-drupal-8-and-search-api-solr/
